README.md

```diff
@@ -32,7 +32,7 @@ Read more on [Getting Started](https://developer.chrome.com/extensions/getstarte
     <tr>
       <td style="vertical-align:top;">
         Cookie Clearer <br>
-        <a href="api/cookies/cookie_clearer"><code>api/cookies/cookie_clearer</code></a>
+        <a href="api/cookies/cookie-clearer"><code>api/cookies/cookie_clearer</code></a>
```

Comment:
should you update the visible text to match the link?

Reply:
D'oh. Fixed!